lua/swenv/api.lua

```diff
@@ -110,12 +115,12 @@ end
 
 local get_pixi_base_path = function()
   local current_dir = vim.fn.getcwd()
-  local pixi_root = Path:new(current_dir) / '.pixi'
+  local pixi_root = Path:new(current_dir):joinpath('.pixi')
 
   if not pixi_root:exists() then
     return nil
   else
-    return pixi_root / 'envs'
+    return pixi_root .. '/envs'
```

Comment:
Why is this better?

Reply:
I think that was a typo from my part, i'll put it back is it was